### PR TITLE
CI: label PRs that have conflicts

### PR DESCRIPTION
**Pull request by Borda**
This will simplify users to navigate what is needed

cc: @EnricoMi

### CODE REVIEW

**Comment by Borda** [comment]
@sfdye @EnricoMi could you pls have look / review :flamingo:

**Comment by Borda** [comment]
@EnricoMi could we include this action to help contributors keep up to date with their PRs? :pray:

File: .github/workflows/label-conflicts.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+on:
+  push: { branches: ["main"] }
+
+jobs:
+  triage-conflicts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mschilde/auto-label-merge-conflicts@v2
```
Comment (Borda) [inline .github/workflows/label-conflicts.yml:8]:
```suggestion
      - uses: mschilde/auto-label-merge-conflicts@v2.0
```

**Comment by Borda** [comment]
this would also need to create a new label `has conflicts`

**Comment by EnricoMi** [comment]
@Borda can we give the workflow a nice name, like `"Label conflicts"`?

**Comment by EnricoMi** [comment]
@Borda looks like the README is outdated:

> Warning: Unexpected input(s) 'MAX_RETRIES', 'WAIT_MS', valid inputs are ['entryPoint', 'args', 'CONFLICT_LABEL_NAME', 'GITHUB_TOKEN']

https://github.com/PyGithub/PyGithub/actions/runs/5590082282/jobs/10219320582#step:3:1

**Comment by Borda** [comment]
I used specific tag, but yes let send update with name and hash

**Pull request merged**